docs/man/man1/kubectl-get.1

```diff
@@ -184,6 +188,9 @@ $ kubectl get replicationcontroller web
 // List a single pod in JSON output format.
 $ kubectl get \-o json pod web\-pod\-13je7
 
+// List a single pod from pod.yaml in JSON output format.
+$ kubectl get \-f pod.yaml \-o json
```

Comment:
Please explain that this just gets the identities of the resources to get from the file(s).